toolchains/riscos/Dockerfile.m4

```diff
@@ -97,3 +147,8 @@ helpers_package(fribidi)
 local_package(libsdl1.2)
 
 helpers_package(sdl-net1.2)
+
+ENV \
+	CFLAGS="" \
+	CXXFLAGS="" \
+	LDFLAGS=""
```

Comment:
I suppose you should unset `ACLOCAL_PATH` and `PKG_CONFIG_LIBDIR` too.
As you seem to have to architectures, maybe something like I did with Android could be useful with a wrapper script which builds each package for both architectures.
Anyway, that could be done later.

Reply:
OK, done.